field_rrule/hooks.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import socket
+import inspect
+
+
+def post_load_hook():
+    """do some trickery to have demo data/model on runbot, but nowhere else"""
```

Comment:
I think it would clutter our repos a lot if modules that don't demo themselves come with demo modules, especially in server-tools where we have many foundational modules that don't do anything by themselves. This is only meant as convenience for the reviewers to be able to test the thing on runbot without having to clone and install the branch locally.